eng/packages/http-client-csharp-mgmt/emitter/src/emitter.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { EmitContext } from "@typespec/compiler";
+import { DecoratorInfo } from "@azure-tools/typespec-client-generator-core";
+
+import {
+  $onEmit as $onMTGEmit,
+  CodeModel,
+  CSharpEmitterOptions,
+  InputModelType,
+  setSDKContextOptions
+} from "@typespec/http-client-csharp";
```

Comment:
Should we use the azure emitter in case more logic gets added there?

Reply:
updated to use `$onAzureEmit` instead of `$onMTGEmit`